import "ent/entidade.proto";

package ent;

enum TipoForma {
  TF_CUBO = 0;
  TF_ESFERA = 1;
  TF_CIRCULO = 2;
  TF_RETANGULO = 3;
  TF_LIVRE = 4;
}

message FormaProto {
  optional TipoForma tipo = 1;
  optional uint32 id = 2;
  optional Cor cor = 3;
  optional Posicao inicio = 4;
  optional Posicao fim = 5;
  repeated Posicao ponto = 6;
}
